### PR TITLE
CORE-11836. VSSolution: Fix MSI_WINETEST compile error with VS10 and VS12

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

JIRA issue: [CORE-11836](https://jira.reactos.org/browse/CORE-11836)

## Proposed changes

- MSVC_IDE: Add an 'include_directories()'

### CODE REVIEW

File: modules/rostests/winetests/msi/CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,10 @@ list(APPEND SOURCE
     suminfo.c
     precomp.h)
 
+if(VS_SOLUTION)
+    # Let rc.exe find custom.dll in its subdirectory, i.e. Debug.
+    include_directories($<TARGET_FILE_DIR:custom>)
```
Comment (ThFabba) [inline modules/rostests/winetests/msi/CMakeLists.txt]: Does anything break if you just do this unconditionally?

Comment (ThFabba) [reply modules/rostests/winetests/msi/CMakeLists.txt]: According to other CMake files in the repo, include_directories should be up between add_definitions and spec2def

File: modules/rostests/winetests/msi/CMakeLists.txt [hunk 2]
```diff
@@ -23,6 +23,10 @@ list(APPEND SOURCE
     suminfo.c
     precomp.h)
 
+if(VS_SOLUTION)
```
Comment (ThFabba) [inline modules/rostests/winetests/msi/CMakeLists.txt]: If you end up needing this check... MSVC_IDE should do the trick. I don't think you need a new variable.

Comment (SergeGautherie) [reply modules/rostests/winetests/msi/CMakeLists.txt]: Right, I assumed cmake `MSVC_IDE` meant its visual interface, whereas it is simply similar to configure.cmd `VS_SOLUTION` build tool.
Done.

File: modules/rostests/winetests/msi/CMakeLists.txt (hunk 1 above)
Comment (SergeGautherie) [reply modules/rostests/winetests/msi/CMakeLists.txt]: Moved up.

I haven't tested this "unconditionally" with other build setups, yet.
While I assume it shouldn't harm, it seems useless to me to enable it when not needed...

**Pull request merged**